automation_tools/utils.py

```diff
@@ -0,0 +1,81 @@
+"""Utilities tasks and functions"""
+from __future__ import print_function
+
+import re
+import sys
+
+from fabric.api import env, run
+
+
+def distro_info():
+    """Task which figures out the distro information based on the
+    /etc/redhat-release file
+
+    A `(distro, major_version)` tuple is returned if called as a function. For
```

Comment:
Double backticks denote inline code in a reStructuredText document. Single backticks denote inline code in a markdown document.